docs/docs/guides/menu.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Controlling the menu
+---
+
+# Controlling the menu
+
+Infrahub allows you to control the menu on the left side of the web interface.
+
+The menu is made up of 2 sections, the bottom section is fixed and cannot be changed. It will list menu items that are related to the working of multiple Infrahub features.
+
+The top section is the section that can be changed, or controlled by the user. The goal of this top section is to list the different types of nodes that you have defined in your schema, in an order and organised in a way that is most relevant for your use case.
+
+By default the top section of the menu will contain an IPAM and Other section. All the nodes defined in your schema, will by default be grouped under the Other section.
```

Comment:
```suggestion
By default the top section of the menu will contain an IPAM and Other section. 
```
After #4716, all elements of the schema will be displayed at the top level by default